externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoContext.scala

```diff
@@ -17,24 +17,9 @@
 
 package org.apache.kyuubi.engine.trino
 
-import java.util.concurrent.atomic.AtomicReference
-
 import io.trino.client.ClientSession
 import okhttp3.OkHttpClient
 
-class TrinoContext(
-    val httpClient: OkHttpClient,
-    val clientSession: AtomicReference[ClientSession]) {
-
-  def getClientSession: ClientSession = clientSession.get
-
-  def setCurrentSchema(schema: String): Unit = {
-    clientSession.set(ClientSession.builder(clientSession.get).withSchema(schema).build())
-  }
-
-}
-
-object TrinoContext {
-  def apply(httpClient: OkHttpClient, clientSession: ClientSession): TrinoContext =
-    new TrinoContext(httpClient, new AtomicReference(clientSession))
-}
+case class TrinoContext(
+    httpClient: OkHttpClient,
+    var clientSession: ClientSession)
```

Comment:
How about we make `clientSession` to TrinoStatement? or somewhere in TrinoOperation? It's not a session level context.

Reply:
I think it's a session level variable. `ClientSession` stores session configs, such as schema and configs which set by `set` command. If we move it to `TrinoOperation `, `use` and `set` will not work.

Reply:
so why do we reset it for each ExcuteStatement?

Reply:
The new configs was store in `StatementClient`, we need to ues it to update session's `ClientSession`.  And `ClientSession` can not update directly, so we reset it for each ExcuteStatement.

Reply:
After `use schema` execute, we can get the new schame by `StatementClient.getSetSchema` and set it to session's `ClientSession`, so the next stetment will run with correct `schema`.

Reply:
I see, then the original AtomicRef makes sense to me.